src/definition/accessors/ISchedulerExtend.ts

```diff
@@ -1,5 +1,10 @@
 import { IProcessor } from '../scheduler';
 
 export interface ISchedulerExtend {
-    registerProcessors(processors: Array<IProcessor>): void;
+    /**
+     * Register processors that can be scheduled to run
+     *
+     * @param {Array<IProcessor>} processors An array of processors
+     */
+    registerProcessors(processors: Array<IProcessor>): Promise<void>;
```

Comment:
Has there been a release with the return type just being `void` and not `Promise<void>`? If so, then this would be a breaking change and the original should be deprecated with a new one being a promise.

Reply:
That was not the case :)

Reply:
Okay, was just making sure a release had not happened yet. :)